recipe/meta.yaml

```diff
@@ -29,7 +30,7 @@ about:
   license_family: OTHER
   # Each data source has a *README.txt file with the license info, that is
   # copied with the datum files. The following CSV file is a summary.
-  license_file: copyright_and_licenses.csv
+  license_file: data/copyright_and_licenses.csv
```

Comment:
We don't need to be updating the version of this file in the recipe.  We can use the one from the source file.

(It just took me awhile to figure that out.)